wagtail/admin/templates/wagtailadmin/workflows/confirm_delete.html

```diff
@@ -6,12 +6,12 @@
     {% include "wagtailadmin/shared/header.html" with title=view.page_title subtitle=view.get_page_subtitle icon=view.header_icon %}
 
     <div class="nice-padding">
-        <p>This workflow is currently in progress on {{ workflow_states_in_progress }} pages. Deleting this workflow will cancel moderation on these page.
-            Are you sure you want to delete this workflow?</p>
+        <p>{% blocktrans %} This workflow is currently in progress on '{{ workflow_states_in_progress }}' pages. Deleting this workflow will cancel moderation on these page.
```

Comment:
Please can you add `trimmed` to the blocktrans 

`{% blocktrans trimmed %}`

Also please remove the added whitespace before ` This` and after `workflow? `

Reply:
Also .I don't think you need the single quotes around the variable usage. This will put literal quotes around the value.

Reply:
yes trimmed shall remove all extra spaces. I did it. I will commit now. Thank you for pointing the trimmed tag out